bazaar_prices.py

```diff
@@ -20,70 +23,78 @@ def connection_info(response):
         sleep(1)
         print("Getting data...")
         sleep(1)
-        print("Data successfully received!")
+        print("Data successfully recived!")
```

Comment:
i think its supposed to be received

Reply:
english isn't my main language and im pretty sure i didnt even touch that line

Reply:
oh